package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "test": "jest --verbose --coverage",
     "size": "size-limit",
-    "prepublishOnly": "copyfiles index.js matcher.js use-location.js static-location.js \"extra/**\" preact/"
+    "prepublishOnly": "copyfiles index.js matcher.{js,d.ts} use-location.{js,d.ts} static-location.{js,d.ts} \"extra/**\" preact/"
```

Comment:
`extra` folder not exists

Reply:
Right, I've removed it in the 2.0 release. @Ty3uK Can you remove `\"extra/**\"` from the command? It is not used anymore.

Reply:
@molefrog should I remove extra from copying?

Reply:
Yes.